src/components/RadioButton/RadioButton.st.css

```diff
@@ -65,6 +65,12 @@
     cursor: pointer;
 }
 
+.wrapperLabelOnly{
```

Comment:
So this can actually be:
```suggestion
.wrapper:not(.withChildren) {
```
right?